src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/ControlChars.vb

```diff
@@ -0,0 +1,22 @@
+' Copyright (c) Microsoft. All rights reserved.
+' Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+Namespace Microsoft.VisualBasic
+    ' Contants for the Control Characters
+    Public NotInheritable Class ControlChars
+
+        Public Const CrLf As String = ChrW(13) & ChrW(10)
+        Public Const NewLine As String = ChrW(13) & ChrW(10)
```

Comment:
Should NewLine be defined in terms of Environment.NewLine?  The latter will be "\r\n" on Windows but "\n" on Unix, and I assume we'd want the same behavior for ControlChars.NewLine?